cmd/fablab/subcmd/cat.go

```diff
@@ -0,0 +1,66 @@
+/*
+	Copyright 2019 NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package subcmd
+
+import (
+	"github.com/openziti/fablab/kernel/fablib"
+	"github.com/openziti/fablab/kernel/model"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(getFilesCmd)
+}
+
+var getFilesCmd = &cobra.Command{
+	Use:   "get-files hostSpec localPath remoteFiles",
```

Comment:
`get-files <hostSpec> <localPath> <remoteFiles>` ... very minor nitpick.

Reply:
Also, can we add a `get` command, which contains the `files` command... rather than a hyphenated `get-files`?

Reply:
Fixed both